src/api.ts

```
@@ -76,13 +109,161 @@
+              endTime,
+            },
+        info: {
+          tests,
+          testsPassed,
+          headers: res.headers,
+          status: res.status,
+          data: res.data,
+          statusText: res.statusText,
+        },
+      };
+      if (world && world.attach) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)